### DISCUSSION TITLE
ansc_064 non aggiornata

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#1137] [Supporto servizi cooperativi]
Buongiorno,
nel changelog 1.34 ho visto questa modifica
[decodifiche] Aggiunto valore in tipo richiedente decodifica ANSC_64 DEC_TIPO_RICHIEDENTE: 5 = Madre e Padre
andando a scaricare il file relativo nelle decodifiche non trovo nulla di tutto ciò

Posso immaginare che:
ID e ORDINAMENTO dovrei valorizzarli a 5
DESCRIZIONE con "Madre e Padre"
DATAINIZIOVALIDITA a 01-01-2000
DATAFINEVALIDITA a 31-12-9999

Chiedo conferma prima di introdurre dati falsi.

grazie

**Comment by annamaraspin** [comment]
si ripete anche nella versione 1.35
sollecito una risposta

**Comment by annamaraspin** [comment]
versione 1.36.4 , documentazione non aggiornata

**Comment by acolosimosogei** [comment]
Buonasera,
confermiamo che è stato aggiunto un ulteriore valore nella tabella, provvediamo ad aggiornare la documentazione.
Grazie,
Assistenza ANSC

**Comment by acolosimosogei** [comment]
Buongiorno, 
la documentazione è stata aggiornata in merito alla segnalazione, procedo pertanto a chiudere la discussion che può essere aperta nuovamente in futuro in caso di ulteriori segnalazioni.

Tante grazie per la segnalazione,
Assistenza ANSC
